src/parsec.rs

```diff
@@ -224,3 +320,100 @@ fn new_rng() -> Box<dyn Rng> {
         Box::new(rng)
     }
 }
+
+#[cfg(all(test, feature = "mock_parsec"))]
+mod tests {
+    use super::*;
+    use crate::{chain::SectionInfo, routing_table::Prefix, xor_name::XorName};
+    use unwrap::unwrap;
+
+    const DEFAULT_MIN_SECTION_SIZE: usize = 4;
+
+    #[test]
+    fn parsec_size_counter() {
```

Comment:
nit pick probably try to group `#[test]` together and not insert helper function between them

Reply:
Hmm I'm not sure. In general yes I agree, but it's nice to keep functions local to where they are used